modules/aws/vpc/master-elb.tf

```diff
@@ -57,6 +31,14 @@ resource "aws_elb" "api_internal" {
     interval            = 5
   }
 
+  # health_check {
```

Comment:
Why are these disabled?

Reply:
An ELB can only have one health_check. We'll need to come up with a different solution for these.

Reply:
> An ELB can only have one `health_check`.

So drop the commented-out code and file an issue?  Do we need to continue to health-check 49500 independently?

Reply:
Can you add this as comment above.

Reply:
Do we like TODOs in the code? If not I'll opt for removing the code altogether

Reply:
49500 doesn't need a health check because Ignition uses a random resolver. If the endpoint is down, Ignition will try another IP address.

Reply:
what if all are down?

Reply:
If they are all down, Ignition keeps retrying.